### DISCUSSION TITLE
How to stop showing bottom statusline when lualine is enabled?

### DISCUSSION THREAD

**Discussion by MoltenMonster** [nvim-lualine/lualine.nvim#1348] [Q&A] [answered]
I have `vim.opt.laststatus = 0` in my config, but it only works when lualine is disabled. I did a bunch of looking up and tried the following: `require('lualine').setup({ options = { globalstatus = true }})`, as well putting `vim.go.laststatus = 0` in `options` and in my config. The only difference I noticed with those is that it would go from showing some text in the statusline, to still showing the statusline just with no text.

The only thing that works is by writing the commands `set laststatus=0` or `lua vim.opt.laststatus=0` after nvim launches. When I omit all settings of `laststatus` in my config and do `verbose set laststatus` after starting nvim up with `nvim -V1`, it shows nothing. It mentions lua when I don't omit them.

**Comment by samuellawerentz** [comment]
I've followed this question in stackoverflow, reddit and finally reached here. Did you find an answer?

**Reply by MoltenMonster** [reply]
I think its caused by a plugin but I didn't bother fixing it yet actually.

**Reply by MoltenMonster** [reply]
@samuellawerentz I found this [discussion](https://github.com/nvim-lualine/lualine.nvim/issues/921) where the dev refuses to stop overriding opts like laststatus, and made this discussion post: https://www.reddit.com/r/neovim/comments/1spl3k3/what_do_you_think_about_plugins_overriding_user/

**Comment by ShreeChaturvedi** [comment, marked as answer]
Call `hide` after your existing `setup()`, then set `laststatus=0`. `setup()` always writes `laststatus` to 2 (3 if `globalstatus` is on), so the `laststatus=0` in your init gets overwritten.

```lua
require('lualine').hide({ place = { 'statusline' } })
vim.opt.laststatus = 0
```

That's in the README under Disabling lualine. It has to run after setup, not inside `options`. `hide` only makes lualine let go of the bar. Drop `laststatus=0` and you'll keep the empty one you already saw, just with no text.
